### DISCUSSION TITLE
No entiendo como hacer las nuevas Katas

### DISCUSSION THREAD

**Discussion by Juan-2003** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#46] [Q&A]
Acabo de ver las nuevas katas, pero no entiendo que hacer. Según yo,  Omar solo nos ha enseñado teoría. Esto lo tenemos que investigar por nuestra cuenta o lo explicara despues?

**Comment by AlejandroMontoyaT** [comment]
Poner en practica lo aprendido en las clases, la clase del dia jueves explica mas a profundidad como podriams realizar el UX y UI,  yo lo estoy trabajando con https://material.io/resources y en Adobe XD me resulto mas facil por que esta en español.

**Comment by doguedogue** [comment]
Con la teoría que tenemos a la mano, podemos ir definiendo como vas a diseñar, a quien va dirigido, etc.
Solo es análisis, diseño y planeación (sin minimizarlo) ponte en el rol y te va ir saliendo punto por punto.
Lo que si, es como usar las herramientas aunque no son obligatorias, pero son muy intuitivas no vas a tener problemas.

**Reply by Juan-2003** [reply]
Osea, por ejemplo. En la primera que es "Requerimientos". Ahi mismo en el archivo de word tenemos que escribir todo lo que pide?

**Comment by Ingrzs** [comment]
Practica 1 rellenar el doc, si ahi partes que no entiendes saltalas, rellenas lo que puedas.
Practica 2 buyer, hazlo , eso se explicó en el live
Práctica 3 público objetivo, hazlo eso también se explicó como es mas no como se hace, usa notion o lo que quieras pero hazlo (investiga cómo se hace)
Practica 4 diagrama ,has un diagrama de todo lo que hara la web 
Practica 5 , haz las interfaces de toda la web
(Si no sabes usas los programas busca tutoriales)

Recomiendo que avanzes las ultimas dos porque conforme lo vas haciendo salen mas cosas 😅 y asi...se necesita tiempo, investigar, etc.
Ya haz hasta donde puedas en lives de hoy preguntemos las dudas y así continuar.

**Comment by DanyVeneno** [comment]
¡Hey! que tal recuerden que  este conocimiento es para la posteridad no se rindan simplemente es un procesos para abordar un proyecto real  
existe una filosofia que dice que " Entre más rapído te equivoques más rapido tienes la solución" te recomiendo veas los videos que se quedan grabados y si utiliza las herramientas que nos dio el MCRodrigo hay algunas que tienen su curva de aprendizaje de todos modos al rato en el live va a dar más feedback saludos buen dia.
![Screenshot_4](https://user-images.githubusercontent.com/89219507/155026551-fb654654-4d21-41ec-82a6-3b4ba329f713.jpg)

**Comment by JuanSalvador73** [comment]
Entonces, ¿lo que hay que entregar es un documento describiendo punto x punto los requerimientos dados, acompañados de un sketch o wireframe de como quedaría en esta etapa temprana de desarrollo el sitio de Abogados?

Saludos

**Reply by BenitoDelaRosaChavez** [reply]
Si hasta donde tengo entendido si,

**Comment by DanyVeneno** [comment]
Heyy que tal saludos Explorers Juan y JuanSalvador.  Si son 5 hay que estar contestando y utilizando las herramientas la que más nos sea adecuadas y lean bien el caso del Abogabot ahi esta la clave y no dudes en buscar en google y youtube y darle una revisada a las herramientas que nos proporcionarón y partan de la pregunta como le harian ustedes con los datos que ya nos dierón.

Como recomendación para que tengan más contexto hay dos talleres en los cuales solo les debes invertir unas 5 hrs. de tu tiempo creen una cuenta en hubspot y busquen el taller de inbound Marketing y otra es Rock University y busquen el taller de Estrategia de  Marketing y confirmen si han solucionado ó no. Saludos buen día
